libs/pages/services/src/lib/feature/page-database-create-feature/step-summary-feature/step-summary-feature.tsx

```diff
@@ -33,11 +33,7 @@ export function StepSummaryFeature() {
   }
 
   const onBack = () => {
```

Comment:
You use only one time the onBack and gotoResources functions on this component, you can directly add them to the props for StepSummary